chart/openfaas/values.yaml

```diff
@@ -24,7 +24,7 @@ faasnetesd:
     periodSeconds: 10
 
 gateway:
-  image: openfaas/gateway:0.9.14
+  image: openfaas/gateway:0.10.0
```

Comment:
Thanks for this PR. FYI your change went out in: https://github.com/openfaas/faas/releases/tag/0.10.2

Reply:
@LucasRoesler FYI